src/main/scala/xerial/sbt/Sonatype.scala

```diff
@@ -58,6 +58,22 @@ object Sonatype extends AutoPlugin {
     pomIncludeRepository := { _ =>
       false
     },
+    credentials ++= {
+      val alreadyContainsSonatypeCredentials = credentials.value.collect { case d: DirectCredentials => d.host == sonatypeCredentialHost.value }.nonEmpty
+      if (!alreadyContainsSonatypeCredentials) {
+        val env = sys.env.get(_)
+        (for {
+          username <- env("SONATYPE_USERNAME").orElse(env("SONATYPE_USER"))
```

Comment:
I think supporting env name variations is convenient, but it's not a good practice. Using only SONATYPE_USERNAME and SONATYPE_PASSWORD is sufficient as documented.